applications/asset_tracker/src/cloud_codec/cloud_codec.h

```diff
@@ -128,16 +147,39 @@ enum cloud_cmd_state {
 	CLOUD_CMD_STATE_TRUE,
 };
 
-struct cloud_command {
-	enum cloud_cmd_group group; /* The group the decoded command belongs to. */
-	enum cloud_cmd_recipient recipient; /* The command's recipient module. */
-	enum cloud_channel channel; /* The command's desired channel. */
-	enum cloud_cmd_type type; /* The command type, the desired action. */
+#define CLOUD_CMD_TYPE_STR_EMPTY		"empty"
+#define CLOUD_CMD_TYPE_STR_ENABLE		"enable"
+#define CLOUD_CMD_TYPE_STR_THRESH_LO	"thresh_lo"
+#define CLOUD_CMD_TYPE_STR_THRESH_HI	"thresh_hi"
+#define CLOUD_CMD_TYPE_STR_INTERVAL		"interval"
+#define CLOUD_CMD_TYPE_STR_COLOR		"color"
+#define CLOUD_CMD_TYPE_STR_MODEM_PARAM	"modemParams"
+
+#define MODEM_PARAM_BLOB_KEY_STR		"blob"
+#define MODEM_PARAM_CHECKSUM_KEY_STR	"checksum"
+
+struct cloud_command_state_value {
 	double value; /* The value to be written to the recipient/channel. */
 	/* The truth value to be written to the recipient/channel. */
 	enum cloud_cmd_state state;
 };
 
+struct cloud_command_modem_params {
+	/* TBD */
```

Comment:
`TBD`? Please elaborate on what's missing.

Reply:
i've added the initial items to this struct, but AGPS is a work in progress (by @jtguggedal).
i can remove the TBD comment if you prefer.

Reply:
I don't mean to remove it, just to elaborate on why it's there. A lone `/* TBD */` says nothing to fellow developers, and if it was me, I'd probably forget after a week why I've added it.

For instance `/* TODO Add AGPS data when ready. */` gives information that we wait for AGPS extensions in the structure. And it's easier for reviewers to keep track on what's going on in the PR.